test/SpokePool.Admin.ts

```diff
@@ -0,0 +1,80 @@
+import { expect } from "chai";
+import { Contract } from "ethers";
+import { ethers } from "hardhat";
+import { SignerWithAddress } from "./utils";
+import { ZERO_ADDRESS } from "@uma/common";
+import { deploySpokePoolTestHelperContracts } from "./SpokePool.Fixture";
+import { depositDestinationChainId } from "./constants";
+
+let spokePool: Contract, erc20: Contract;
+let admin: SignerWithAddress;
+
+describe("SpokePool Admin Functions", async function () {
+  beforeEach(async function () {
+    [admin] = await ethers.getSigners();
+    ({ spokePool, erc20 } = await deploySpokePoolTestHelperContracts(admin));
+  });
+  it("Whitelist token path", async function() {
+    // Cannot set destination token to zero address.
+    await expect(spokePool
+        .connect(admin)
+        .whitelistRoute(
+          erc20.address,
+          ZERO_ADDRESS,
+          true,
+          depositDestinationChainId
+        )).to.be.reverted;
+
+    await expect(spokePool
+      .connect(admin)
+      .whitelistRoute(
```

Comment:
is this listed? I feel that this can be done on fewer lines, no? do you have the prettier vs code plug in installed: https://github.com/prettier/prettier-vscode?

this will let you lint without needing to run prettier, using the config that I set when creating the repo.

Reply:
I think the main issue is that I was erroneously relying on CI to lint like it does in `protocol` repo. So I will manually lint and commit